common.sh

```diff
@@ -1,31 +1,59 @@
 
 CACHE=/var/cache/preupgrade
+# Directory with logs gathered by preupgrade-assistant
```

Comment:
this and whole section: it's impossible to know for sure which comments belongs to which variable!